drivers/spi/spi_silabs_eusart.c

```diff
@@ -37,11 +64,26 @@ struct spi_silabs_eusart_config {
 	const struct silabs_clock_control_cmu_config clock_cfg;
 	uint32_t clock_frequency;
 	const struct pinctrl_dev_config *pcfg;
+	volatile uint8_t mosi_overrun_val;
+#if defined(CONFIG_SPI_SILABS_EUSART_DMA)
+	const bool is_dma_enabled_instance;
+#endif
```

Comment:
Do not duplicate the information. This field is redundant with `dma_channel_rx`/`dma_channel_tx`.